tensorflow/compiler/xla/client/computation_builder.h

```diff
@@ -211,9 +211,11 @@ class ComputationBuilder {
   //
   // Note that "limit" means up-to-but-not-including; i.e. [start, limit) in 1D
   // range notation.
+  // The stride parameter determines the stride over the slice
```

Comment:
ok.  i'm on holiday next week, but I will try to get it sorted out this weekend.  shouldn't be hard to add the rev+offset to the tf2xla and remove from the elemental emitter.